libraries/AP_OpticalFlow/AP_OpticalFlow_Pixart.cpp

```diff
@@ -166,25 +166,40 @@ bool AP_OpticalFlow_Pixart::setup_sensor(void)
 
     integral.last_frame_us = AP_HAL::micros();
 
-    _dev->register_periodic_callback(2000, FUNCTOR_BIND_MEMBER(&AP_OpticalFlow_Pixart::timer, void));
+    _dev->register_periodic_callback(5000, FUNCTOR_BIND_MEMBER(&AP_OpticalFlow_Pixart::timer, void));
```

Comment:
why change the loop rate?

Reply:
> why change the loop rate?

It was explained in the description of this PR https://github.com/ArduPilot/ardupilot/pull/10033#issue-239038421

I'm also curious that why such a high loop rate in the first place? Is this really necessary?